wiki/Configuration:-Input.md

```diff
@@ -240,11 +240,13 @@ input {
 
 Makes the mouse warp to newly focused windows.
 
-X and Y coordinates are computed separately, i.e. if moving the mouse only horizontally is enough to put it inside the newly focused window, then it will move only horizontally.
+By default, X and Y coordinates are computed separately, i.e. if moving the mouse only horizontally is enough to put it inside the newly focused window, then it will move only horizontally. 
+If force-center is set, the mouse will always warp to the center of the focused window. 
 
 ```kdl
 input {
     warp-mouse-to-focus
+    // or warp-mouse-to-focus { force-center; }
```

Comment:
Instead of this, it should be a knuffel property, so `warp-mouse-to-focus force-center=true`